suricata.yaml.in

```diff
@@ -157,6 +157,7 @@ outputs:
             # metadata: no             # enable inclusion of app layer metadata with alert. Default yes
             # http-body: yes           # Requires metadata; enable dumping of HTTP body in Base64
             # http-body-printable: yes # Requires metadata; enable dumping of HTTP body in printable format
+            # flow-ip: no              # Requires metadata; enable dumping of IP info in flow subobject
```

Comment:
I wonder if we should just do this by default for alerts with metadata is yes. One less toggle to worry about, and alerts are outnumbered by far by other event types, so I dont think its much of a performance consideration.

Reply:
Would be really good with me.

Reply:
I can resubmit it like that if you want.